common/focuses/wc_magic_lifestyle_focuses.txt

```diff
@@ -0,0 +1,1257 @@
+﻿light_magic_focus_1 = {
+	lifestyle = light_magic_lifestyle
+
+	is_valid = {
+		can_openly_learn_light_magic_trigger = yes
+	}
+
+	modifier = { }
+
+	desc = {
+		desc = light_magic_focus_1_desc
+		desc = line_break
+	}
+
+	auto_selection_weight = {
+		value = wc_magic_lifestyle_auto_selection_weight_base_value
+
+		# Base
+		if = {
+			limit = { MAGIC_magic_lifestyle_auto_selection_weight_add_base_trigger = { MAGIC = light } }
+			add = wc_magic_lifestyle_auto_selection_weight_add_to_base_value
+		}
+
+		# Personality Traits
+		# TODO
+
+		# Bad Magic Traits
+		if = {
+			limit = {
+				OR = {
+					has_trait = magic_bad
+					has_trait = magic_bad_noninheritable
+				}
+			}
+			multiply = 0
+		}
```

Comment:
You could make it scripted value too.
Reminder, it's possible to use `if` in script values.

Reply:
Done